fvt/fvtclient.go

```diff
@@ -425,6 +425,18 @@ func (fvt *FVTClient) RunKfsInference(req *inference.ModelInferRequest) (*infere
 	return grpcClient.ModelInfer(ctx, req)
 }
 
+func (fvt *FVTClient) RunKsModelMetadata(req *inference.ModelMetadataRequest) (*inference.ModelMetadataResponse, error) {
```

Comment:
Nit: maybe RunKfsModelMetadata to be consistent?

Reply:
Initially wanted to start using the abbreviation for the rename, but, sure, I'll keep it consistent for now. Maybe at some point, we can do a mass rename.

Reply:
Yeah, I thought you might have ks for KServe to be different from ksf. Agreed, at some point, we should rename all kfs.